omnibus/package-scripts/agent-deb/prerm

```diff
@@ -160,7 +137,7 @@ remove_apm_remote_config()
 
 stop_agent
 deregister_agent
-remove_custom_integrations
+${INSTALL_DIR}/embedded/bin/python "${INSTALL_DIR}/python-scripts/prerm.py" "${INSTALL_DIR}"
```

Comment:
nit: it might be worth to wrap this in a function to keep it readable and self explanatory

Reply:
I'd prefer keep it simple with just one line. Maybe adding a comment with the purpose?